apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -136,4 +144,51 @@ export const siteRouter = router({
 
       return input
     }),
+  setSiteConfigAdmin: protectedProcedure
```

Comment:
nit: i think this might be better named as `setSiteConfigByAdmin` or `setAdminSiteConfig` but very minor

Reply:
Updated in 31dcf08f43695cdc78c70983cff0df8d9a24e9dd.